tests/acceptance/integration-test.js

```diff
@@ -39,6 +39,19 @@ module('Acceptance | Intersection Observer', function(hooks) {
     assert.ok(find('.my-component.bottom.active'), 'component is active');
   });
 
+  test('Tagless component moves to active when scrolled into viewport', async function(assert) {
+    assert.expect(1);
+
+    await visit('/');
+  
+    assert.ok(find('.tagless-component.inactive'), 'component is inactive');
```

Comment:
I think based on your template, would would want to `assert.notOk(find('.tagless-component > .active'))` and then `assert.ok` when it is scrolled into view.  What do you think?